src/Compilers/VisualBasic/Portable/Symbols/Source/SourceNamedTypeSymbol.vb

```diff
@@ -99,6 +100,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
 #Region "Completion"
         Protected Overrides Sub GenerateAllDeclarationErrorsImpl(cancellationToken As CancellationToken)
+#If DEBUG Then
+            EnsureAllHandlesAreBound()
+#End If
```

Comment:
The ```EnsureAllHandlesAreBound``` invocation is redundant here. The next line ```MyBase.GenerateAllDeclarationErrorsImpl``` binds all handles clauses as well. Executing ``EnsureAllHandlesAreBound``` allows to assert that it is not missing handles clauses.